openstates/settings.py

```diff
@@ -3,6 +3,9 @@
 
 # settings for realtime flag
 S3_REALTIME_BASE = os.environ.get("S3_REALTIME_BASE")  # e.g 's3://realtime-bucket'
```

Comment:
Should we keep the S3 bucket definition if we'll be writing to SQS instead?

Reply:
Yes for now, we are persisting the actuals files for a while. The actual data isn't sent to  SQS yet, just the path to the file on s3. 

cc: @johnseekins @bfossen-ce